README.md

```diff
@@ -34,5 +36,9 @@ See the [contributing guide](CONTRIBUTING.md) for detailed instructions on how t
 
 ### Miscellaneous
 
+
+## Dating Apps
```

Comment:
I don't think we need this section since `Dating Apps` don't really fit in with the idea of a *classroom* toolkit.